open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-encrypted-file-store-connector/pom.xml

```diff
@@ -44,5 +44,27 @@
         </dependency>
 
     </dependencies>
+    <build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+                <execution>
+                    <id>analyze</id>
+                    <goals>
+                        <goal>analyze-only</goal>
+                    </goals>
+                    <configuration>
+                        <ignoredUnusedDeclaredDependencies combine.children="append">
+                            <!-- ui admin  - false positives from dependency checker -->
+                            <ignoredUnusedDeclaredDependency>org.odpi.egeria:ui-admin-services-api:*</ignoredUnusedDeclaredDependency>
```

Comment:
I wouldn't expect this to be dynamically loaded (ie by spring component scan) since api normally represents our common classes for server/client to share. Have you checked for unnecessary imports? Can you confirm this is dynamically loaded?

Reply:
See patch reference below -